api/src/main/java/org/openmrs/module/labintegration/api/hl7/messages/OrderControl.java

```diff
@@ -0,0 +1,5 @@
+package org.openmrs.module.labintegration.api.hl7.messages;
+
+public enum OrderControl {
+	NW, CA
```

Comment:
Maybe you could wrap those shortcuts into more easy-to-read form? See other occurences